BuildingSync.xsd

```diff
@@ -432,7 +432,7 @@
         </xs:annotation>
         <xs:simpleType>
           <xs:restriction base="xs:string">
-            <xs:enumeration value="3.0.0"/>
+            <xs:enumeration value="3.0.0-pr1"/>
```

Comment:
Probably fine for now, but I think we should remove this when we make the official 3.0 release

Reply:
> Probably fine for now, but I think we should remove this when we make the official 3.0 release

💯% agreed. I removed all the pr.'s from v2 schema.